api/default_networks_test.go

```diff
@@ -25,68 +25,42 @@ func TestBuildDefaultNetworks(t *testing.T) {
-
-	require.True(t, strings.Contains(actualNetworks[1].RPCURL, infuraToken))
-
-	require.Equal(t, sepoliaChainID, actualNetworks[2].ChainID)
-
-	require.True(t, strings.Contains(actualNetworks[2].RPCURL, poktToken))
-	require.True(t, strings.Contains(actualNetworks[2].FallbackURL, infuraToken))
-	require.True(t, strings.Contains(actualNetworks[2].DefaultRPCURL, stageName))
-	require.True(t, strings.Contains(actualNetworks[2].DefaultFallbackURL, stageName))
-	require.True(t, strings.Contains(actualNetworks[2].DefaultFallbackURL2, stageName))
-
-	require.Equal(t, optimismChainID, actualNetworks[3].ChainID)
-
-	require.True(t, strings.Contains(actualNetworks[3].RPCURL, poktToken))
-	require.True(t, strings.Contains(actualNetworks[3].FallbackURL, infuraToken))
-	require.True(t, strings.Contains(actualNetworks[3].DefaultRPCURL, stageName))
-	require.True(t, strings.Contains(actualNetworks[3].DefaultFallbackURL, stageName))
-	require.True(t, strings.Contains(actualNetworks[3].DefaultFallbackURL2, stageName))
-
-	require.Equal(t, optimismGoerliChainID, actualNetworks[4].ChainID)
-
-	require.True(t, strings.Contains(actualNetworks[4].RPCURL, infuraToken))
-	require.Equal(t, "", actualNetworks[4].FallbackURL)
-
-	require.Equal(t, optimismSepoliaChainID, actualNetworks[5].ChainID)
-
-	require.True(t, strings.Contains(actualNetworks[5].RPCURL, poktToken))
-	require.True(t, strings.Contains(actualNetworks[5].FallbackURL, infuraToken))
-	require.True(t, strings.Contains(actualNetworks[5].DefaultRPCURL, stageName))
-	require.True(t, strings.Contains(actualNetworks[5].DefaultFallbackURL, stageName))
-	require.True(t, strings.Contains(actualNetworks[5].DefaultFallbackURL2, stageName))
-
-	require.Equal(t, arbitrumChainID, actualNetworks[6].ChainID)
-
-	require.True(t, strings.Contains(actualNetworks[6].RPCURL, poktToken))
-	require.True(t, strings.Contains(actualNetworks[6].FallbackURL, infuraToken))
-	require.True(t, strings.Contains(actualNetworks[6].DefaultRPCURL, stageName))
-	require.True(t, strings.Contains(actualNetworks[6].DefaultFallbackURL, stageName))
-	require.True(t, strings.Contains(actualNetworks[6].DefaultFallbackURL2, stageName))
-
-	require.Equal(t, arbitrumGoerliChainID, actualNetworks[7].ChainID)
-
-	require.True(t, strings.Contains(actualNetworks[7].RPCURL, infuraToken))
-	require.Equal(t, "", actualNetworks[7].FallbackURL)
-
-	require.Equal(t, arbitrumSepoliaChainID, actualNetworks[8].ChainID)
-
-	require.True(t, strings.Contains(actualNetworks[8].RPCURL, poktToken))
-	require.True(t, strings.Contains(actualNetworks[8].FallbackURL, infuraToken))
-	require.True(t, strings.Contains(actualNetworks[8].DefaultRPCURL, stageName))
-	require.True(t, strings.Contains(actualNetworks[8].DefaultFallbackURL, stageName))
-	require.True(t, strings.Contains(actualNetworks[8].DefaultFallbackURL2, stageName))
+	ignoreDefaultRPCURLCheck := false // TODO: used just because of Goerli, remove once we remove Goerli from the default networks
```

Comment:
It's fine to leave it if removing Goerli now will introduce too much complexity in this PR, there's 0 reason to have it since none of the providers we use support it still.

Reply:
Here is a new PR that removes Goerli from the code https://github.com/status-im/status-go/pull/5945